### ISSUE TITLE
icon for closing the application

### ISSUE THREAD

**Issue by nnako** [mPyKen/ScreenAreaShare#21]
When starting `ScreenAreaShare`, both windows (for capture and for render) open at their user-desired locations on any screen, respective to their starting coordinates and size. Both screens will always be fully visible (as long as they are not bigger than the hosting screen). Both windows open on top of all other windows (which is standard for new applications on Windows OS).

Now, after presenting, I'd like to easily close the entire application (including the capture and the render window). As far as I know, there is no icon to do so. Neither on the capture window nor on the render window. So, I have to do the following in order to quit the application:

- scan the task row for a fitting console which I used to start the `ScreenAreaShare` application
- bring that console to the front by clicking that icon
- pressing `<CTRL> + <C>` to send a `sigint` signal
-> this closes capture and render windows
- press `<Y> + <RETURN>` to end the batch process and close the console

**Would it be possible to operate a small "close" icon on the top right of each window (capture and render windows) to be able to terminate the application?**

If this is a bad idea, please tell me. Or is there already a way to comfortably quit the application's operation without fiddling with the console window (somewhere behind some other windows)? Maybe a hidden hotkey?

**Comment by mPyKen** [comment]
You can close the application by hitting ESC or alt F4 when the rendering window is in focus

**Comment by nnako** [comment]
So, when intending to close the application, as soon as the render window is not in front, you would first have to

- find the render window in the icon bar
- click it there to be in front, then
- press `<ESC>`

But it could be a benefit, if there was a (even small) icon `<X>` at the top right of the capture window. As the capture window is always on top, this could be the fastest way to deliberately close the application without even knowing about the current location of the render window.

What do you think?

**Comment by mPyKen** [comment]
True, having a close button on the capture window would be a nice feature!
